Sindri-Labs/sindri-resources
# Patch mainnet circuit (halo2)

This PR locks the `tokio` dependency to a version so it works with the specified rust version `1.69.0-nightly`. `tokio` has newer versions that require upgrading the rust version.

## Testing Instructions:
Submit both of the following to the frontend circuit upload to ensure they were re-compressed and patched properly:
- [ ] `circuit_database/halo2/axiom-v0.3.0/mainnet_header.tar.gz`
- [ ] `circuit_database/halo2/axiom-v0.3.0/mainnet_header.zip`
